src/main/java/mate/academy/spring/boot/dto/BookDto.java

```diff
@@ -11,6 +11,8 @@ public class BookDto {
     private String description;
     private String coverImage;
 
+    private boolean deleted;
```

Comment:
If it's response BookDto, stick to the task description 
Example of response body:
```
[
  {
    "id": 1,
    "title": "Sample Book 1",
    "author": "Author A",
    "isbn": "9781234567897",
    "price": 19.99,
    "description": "This is a sample book description.",
    "coverImage": "http://example.com/cover1.jpg"
  },
  {
    "id": 2,
    "title": "Sample Book 2",
    "author": "Author B",
    "isbn": "9789876543210",
    "price": 24.99,
    "description": "Another sample book description.",
    "coverImage": "http://example.com/cover2.jpg"
  }
]

```

Also, you can use @Data annotation to avoid generating get and set methods, or use java record